lib/constructs/enumeration.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+class Enumeration {
+  constructor(ctx, idl) {
+    this.ctx = ctx;
+    this.idl = idl;
+    this.name = idl.name;
+    this.str = null;
+  }
+
+  generate() {
+    const values = new Set(this.idl.values);
+    if (!this.ctx.options.suppressErrors && values.size !== this.idl.values.length) {
```

Comment:
Per @Sebmaster it looks like we're not using suppressErrors for this type of thing, only for things that are "unavoidable" like not having every interface type implemented.